libs/entitlement/tools/public-callers-finder/README.md

```diff
@@ -0,0 +1,40 @@
+This tool scans the JDK on which it is running. It takes a list of methods (compatible with the output of the `securitymanager-scanner` tool), and looks for the "public surface" of these methods (i.e. any class/method accessible from regular Java code that calls into the original list, directly or transitively).
+
+It acts basically as a recursive "Find Usages" in Intellij, stopping at the first fully accessible point (public method on a public class).
```

Comment:
The main question I'd have at this point is: how do we find all the call sites?
- What is the scope of the scan? Every method in the entire class library?
- What are we scanning for? If we're looking at a method `C.m` are we considering calls to `S.m` where `S` is a supertype of `C`?

These questions could be answered by looking in the code, of course, but if they could be concisely answered here, I think that would be useful.